### PR TITLE
add composio to ecosystem page

### PR DESCRIPTION
**Pull request by Karthikeya-Meesala**
## Why are these changes needed?
- Hey @ekzhu, We'd love to get Composio featured on Autogen Ecosystem. Please review the PR! Thanks!

### CODE REVIEW

**Review by ekzhu — APPROVED** [review]

File: website/docs/ecosystem/img/ecosystem-composio.png
Comment (ekzhu) [inline website/docs/ecosystem/img/ecosystem-composio.png:1]: There is a recent change to track all images with lfs. In your branch, can you remove the image and add it back again to make sure it is tracked by lfs. You can check by using `git lfs ls-files --all`

**Review by ekzhu — CHANGES_REQUESTED** [review]

File: website/docs/ecosystem/img/ecosystem-composio.png
Comment (Karthikeya-Meesala) [reply website/docs/ecosystem/img/ecosystem-composio.png:1]: [Done](https://github.com/Karthikeya-Meesala/autogen/commit/dc83705cc3193a418f237741b5ba578ad120da72) ✅

File: .gitattributes [hunk 1]
```diff
@@ -1,3 +1,4 @@
 *.png filter=lfs diff=lfs merge=lfs -text
 *.jpg filter=lfs diff=lfs merge=lfs -text
 *.jpeg filter=lfs diff=lfs merge=lfs -text
+ecosystem-composio.png filter=lfs diff=lfs merge=lfs -text
```
Comment (ekzhu) [inline .gitattributes:4]:
```suggestion
```

**Review by ekzhu — APPROVED** [review]

File: website/docs/ecosystem/composio.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Composio
+
+![Composio Logo](img/ecosystem-composio.png)
+
+Composio empowers AI agents to seamlessly connect with external tools, Apps, and APIs to perform actions and recieve triggers. With built-in support for AutoGen, Composio enables the creation of highly capable and adaptable AI agents that can autonomously execute complex tasks and deliver personalized experiences.
```
Comment (ekzhu) [inline website/docs/ecosystem/composio.md:5]:
```suggestion
Composio empowers AI agents to seamlessly connect with external tools, Apps, and APIs to perform actions and receive triggers. With built-in support for AutoGen, Composio enables the creation of highly capable and adaptable AI agents that can autonomously execute complex tasks and deliver personalized experiences.
```

**Pull request merged**